### PR TITLE
Add list$ docs

### PR DESCRIPTION
**Pull request by kamil-mech**
https://github.com/senecajs/senecajs.org/issues/53

I'm open to suggestions :)

Should I update https://github.com/senecajs/seneca/tree/master/doc/examples with source?

### CODE REVIEW

File: src/pages/tutorials/understanding-query-syntax.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+layout: main.html
+---
+
+# Understanding Query Syntax
+
+Each seneca store has a set of utility functions. One of them is `list$`, which is used to query for a very specific selection of data.
+
+Note: An awesome npm module <a href="https://www.npmjs.com/package/lodash">lodash</a> is used in examples here for neat display of results (`_` variable).
```
Comment (mcdonnelldean) [inline src/pages/tutorials/understanding-query-syntax.md]: bold `Note:`

File: src/pages/tutorials/understanding-query-syntax.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+- [Combine](#wp-combine)
+
+
+<a name="wp-sample-entities"></a>
+## Sample Entities
+
+For the purpose of this example, a small set of manual entries needs to be created(so that we have something to select from). This can be easily achieved using chaining. Note only one callback is needed - at the very end. Each `make$()` is needed to make sure instances are unique.
+
+``` js
+var thing = seneca.make$('thing')
```
Comment (mcdonnelldean) [inline src/pages/tutorials/understanding-query-syntax.md]: Use person and actual fields, it will be easier to read then f1, f2, etc

**Comment by mcdonnelldean** [comment]
@kamil-mech Awesome first cut! Love it, can we use person throughout, it will be easier to understand. Once done I would update the doc with the code, it keeps everything tidy then.

**Comment by kamil-mech** [comment]
Thank You!

Anything else @mcdonnelldean? :)

**Comment by mcdonnelldean** [comment]
@kamil-mech Nope, just update the example on seneca. Looks super here. I'll merge.

**Pull request merged**